server.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
Do we not want to use TypeScript for the server?

Reply:
Hm, I never thought of that. o_o I guess I always just used the boilerplate custom `server.js` from the Next.js docs. I'll look into it!